addons/website_event_booth/models/event_type.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class EventType(models.Model):
+    _inherit = 'event.type'
+
+    booth_menu = fields.Boolean(
+        string='Booths on Website', compute='_compute_booth_menu',
+        readonly=False, store=True)
+
+    @api.depends('website_menu')
+    def _compute_booth_menu(self):
+        for event_type in self:
+            event_type.booth_menu = event_type.website_menu
```

Comment:
what's the point of this ? Why not put a label in the views on website_menu field instead ? (or relabel the website_menu field in event_type view)